.github/workflows/shoutout.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow is a test to see if we can write a working workflow
+# Author: Matt Sarmiento <msarmiento3@usfca.edu>
+
+name: shout-out 
+on: [fork, watch]
+jobs: 
+  greeting:
+    runs-on: [ubuntu-latest, macos-latest]
+    steps:
+    - name: Hello world
+      with:
+        who-to-greet:  "Fyodor"
```

Comment:
Nice work! Just one quick fix--looks like you're missing a `run` or `uses` parameter here, which is causing the workflow to fail.

Reply:
Got it! Thanks for pointing that out 🙃